storybook/backgrounds.ts

```diff
@@ -13,3 +13,14 @@ export const backgrounds = [
   { name: "Orange 500", value: tokens.kz.color.orange["500"] },
   { name: "Red 500", value: tokens.kz.color.red["500"] },
 ]
+export const cssVarBackgrounds = [
+  { name: "Neutral-100 (White)", value: tokens["kz-var"].color.neutral["100"] },
```

Comment:
👀 The naming consistency is a bit of a bother. Could you change the names to `Neutral 100` etc both in your changes and here: https://github.com/cultureamp/kaizen-design-system/pull/1741/commits/000b95549ad51f383d44e3810a1aca5f3277d93b#diff-b3e4a28d9c367ace4d3bc1ebf576bc4ee1bb1927a3d141a2cf4386224dde60c0R6

Reply:
Yeah for sure, i guess i was thinking people might be thinking "where did white go?" or might not know it exists.

Reply:
Probably just something to think about, we refer to "white" in many places where changing to "neutral-100" isn't the nicest e.g:
![image](https://user-images.githubusercontent.com/7651503/124555161-1a16a480-de7a-11eb-9156-b92d0ea2090d.png)

Reply:
Apologies, I was referring to the mixed-use of snake-case and sentence case. Totally fair (and good) to include `Neutral 100 (white)` as most people will continue to refer to it as white.

Reply:
Ooooohhh lollll